src/aat/resources/features/F-044/S-579-Prerequisite_Solicitor_3_Token_For_Update_Case.td.json

```diff
@@ -30,7 +30,7 @@
     "_extends_": "Common_200_Response",
     "headers": {
       "Content-Encoding": "gzip",
-      "Content-Type": "application/json;charset=UTF-8",
+      "Content-Type": "application/json",
```

Comment:
not for this PR but I was thinking whether it's possible to have for the headers a common header or few common headers so that when similar changes happen we don't have to update all these files

Reply:
Yes, this header should have been in a common one that’s already in place.
Probably was missed initially.